common/src/main/java/com/wynntils/wynn/utils/WynnBossBarUtils.java

```diff
@@ -7,38 +7,45 @@
 import com.wynntils.mc.utils.ComponentUtils;
 import com.wynntils.mc.utils.McUtils;
 import com.wynntils.utils.Pair;
+import java.util.Collection;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import net.minecraft.client.gui.components.LerpingBossEvent;
 
 public class WynnBossBarUtils {
-    public static final ManaBank NO_MANA_BANK = new ManaBank(0, 0, -1);
-    public static final BloodPool NO_BLOOD_POOL = new BloodPool(0, -1);
-    public static final AwakenedBar NO_AWAKENED_PROGRESS = new AwakenedBar(0, 0, -1);
-    public static final Focus NO_FOCUS = new Focus(0, 0, -1);
+    public static final BarProgress NO_MANA_BANK = new BarProgress(0, 0, -1);
+    public static final BarProgress NO_BLOOD_POOL = new BarProgress(0, 0, -1);
+    public static final BarProgress NO_AWAKENED_PROGRESS = new BarProgress(0, 0, -1);
+    public static final BarProgress NO_FOCUS = new BarProgress(0, 0, -1);
 
     private static final Pattern BLOOD_POOL_PATTERN = Pattern.compile("§cBlood Pool §4\\[§c(\\d+)%§4\\]");
     private static final Pattern MANA_BANK_PATTERN = Pattern.compile("§bMana Bank §3\\[(\\d+)/(\\d+)§3\\]");
     private static final Pattern AWAKENED_PROGRESS_PATTERN = Pattern.compile("§fAwakening §7\\[§f(\\d+)/(\\d+)§7]");
     private static final Pattern FOCUS_PATTERN = Pattern.compile("§eFocus §6\\[§e(\\d+)/(\\d+)§6]");
 
-    private static Pair<LerpingBossEvent, Matcher> getLerpingBossEvent(Pattern titlePattern) {
+    private static Pair<LerpingBossEvent, Matcher> getLerpingBossEvent(Pattern titlePattern, boolean cancel) {
```

Comment:
There might be a better name for `cancel`. Perhaps `disableVanillaRendering`? (or anything else that fits better)

Reply:
That would likely be better